packages/react-pdf/src/elements/Document.js

```diff
@@ -20,6 +27,23 @@ class Document {
     this.children.slice(index, 1);
   }
 
+  addMetaData() {
+    const { title, author, subject, keywords } = this.props;
+    // The object keys need to start with a capital letter by the PDF spec
+    if (title) {
+      this.root.info.Title = title;
+    }
+    if (author) {
+      this.root.info.Author = author;
+    }
+    if (subject) {
+      this.root.info.Subject = subject;
+    }
+    if (keywords) {
+      this.root.info.Keywords = keywords;
+    }
```

Comment:
Are these 4 the only type of metadata possible? (I don't actually know 😅 )
Also. is there a problem if some metadata keeps set as`null` value? I'm asking if we can get rid of the `if`s statements, or maybe replacing them for `title || undefined`. Maybe I'm being a bit nitpicking here haha sorry

Reply:
Don't worry. I answer myself 😅 . Didn't worked with the way I suggested